go.mod

```diff
@@ -86,7 +86,7 @@ require (
 	go.opentelemetry.io/otel/sdk v0.20.0
 	go.opentelemetry.io/otel/trace v0.20.0
 	go.opentelemetry.io/proto/otlp v0.7.0
-	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83
+	golang.org/x/crypto v0.0.0-20201216223049-8b5274cf687f
```

Comment:
this is backleveling the library from 20210220033148 to 20201216223049

the current version should contain all prior fixes, right?

/hold

Reply:
Oh, it looks like something is wrong. Let me check later.

Reply:
You are right. 1.21 crypto package version contain the fix  and I should choose  apply this change to 1.20. Update.